test/ring/middleware/test/refresh.clj

```diff
@@ -1,6 +1,88 @@
 (ns ring.middleware.test.refresh
-  (:use clojure.test
-        ring.middleware.refresh))
+  (:require
+   [clojure.java.io :as io]
+   [clojure.test :refer [are deftest testing]]
+   [ring.middleware.refresh :as sut]))
 
-(deftest test-wrap-refresh
-  (is (true? false)))
\ No newline at end of file
+(defn create-success-html-handler [body]
+  (constantly {:body body, :headers {"Content-Type" "text/html"}, :status 200}))
```

Comment:
Can you swap the positions of `:status 200` and `:body body`? As a convention, I tend to order them in the order they appear in a HTTP request.

Reply:
This could also be written as:

```clojure
(defn- create-test-handler [body]
  (-> (constantly {:status 200
                   :headers {"Content-Type" "text/html"}
                   :body body})
      (wrap-refresh))
```

As I don't think we'd ever need a handler without the middleware we're testing attached.

Reply:
Done. I didn't use parens around the arg-less `wrap-refresh` function call. If you prefer these, I can certainly add them.